src/content/guidelines/accessibility/color.md

```diff
@@ -30,7 +33,7 @@ Color blindness involves difficulty in perceiving or distinguishing between colo
 | Monochromacy | All colors |  
 
 ### Designing for color blindness
-Don't rely on color alone to convey meaning. This includes conveying information, indicating an action, prompting the user for a response, or distinguishing one visual element from another. When designing with color, it's always good to use a color-blind simulator to ensure compliance. If you're working in Sketch, we recommend the [Stark](http://www.getstark.co/) plugin.  
+Don't rely on color alone to convey meaning. This includes conveying information, indicating an action, prompting the user for a response, or distinguishing one visual element from another. When designing with color, it my help to use a color-blind simulator to review visibility of content. If you're working in Sketch, we recommend the [Stark](http://www.getstark.co/) plugin.  
```

Comment:
`it my help` -> `it might help` potentially?

Reply:
Oh, thank you, @joshblack - so how do I now make edits and complete the pull. Do I cancel the PR and then make edits and then do another?

Reply:
@bocampbell any commits that you push up on this branch will be reflected in the PR! I think you can also make edits in the PR directly through the GitHub UI if you prefer 😄